operator/Makefile

```diff
@@ -144,7 +144,7 @@ KUSTOMIZE = $(PROJECT_DIR)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
 	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.8.7)
 
-OPERATOR_SDK_VERSION = 1.14.0
```

Comment:
We generally go through the upgrade checklist when bumping operator sdk version...

Reply:
reverted